docs/getting-started/defi-practice.md

```diff
@@ -3,25 +3,70 @@
 
 Decentralized Finance (DeFi) is revolutionizing the finance industry and bringing boundless opportunities to the finance world. However, at the same time it has witnessed millions of funds being hacked. We believe it is critical for us to play a part to minimize such incidents as we place security as a top priority. With that, we have compiled a list of best practices to develop secure smart contracts for the Cronos ecosystem. This is by no means an exhaustive list but we believe it is a good foundation and minimum security baseline for any smart contract project.
 
-## Secure Coding Practices:
+## Secure Coding Practices
 
+### Design
+#### Documentation
```

Comment:
Could we use at max 3 levels, #, ## and ### because 4 don't work on gitbook

Reply:
Sure, thanks for brining it up.